app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -319,18 +321,23 @@ fun ParkingDetailsScreen(
                         colorLevel = ColorLevel.PRIMARY,
                         testTag = "ShowInMapButton")
 
-                    Button(
-                        text = stringResource(R.string.card_screen_report),
-                        onClick = {
-                          Toast.makeText(
-                                  context,
-                                  "A report system will be added to the app later",
-                                  Toast.LENGTH_LONG)
-                              .show()
-                        },
-                        modifier = Modifier.fillMaxWidth(),
-                        colorLevel = ColorLevel.ERROR,
-                        testTag = "ReportButton")
+                    if (userViewModel.currentUser.value != null) {
```

Comment:
Just making sure, those hard coded values, it's because right now we dont have a report screen ?

Reply:
Exactly! No real way to create a custom report yet (this is also why I had to ship this entire massive PR out at once) so this is supposed to check both that the subcollection exists and that it does its job